### ISSUE TITLE
How to add noise in embedding layer？

### ISSUE THREAD

**Issue by yysirs** [UKPLab/sentence-transformers#563]
I want to improve the generalization of the model

**Comment by nreimers** [comment]
Dropout is used in BERT etc.

What you could do is to drop out whole word or word-pieces from the input. But this would require that you modify your training data so that the sentences are incomplete (some random words are deleted)

**Comment by Sauce16** [comment]
hi @yysirs could you share me the code? coz I am new and I am not sure how to add noise in the embedding layer
